src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -152,5 +152,10 @@
         <service id="translation.extractor" class="%translation.extractor.class%"/>
 
         <service id="translation.writer" class="%translation.writer.class%"/>
+
+        <service id="translation.warmer" class="Symfony\Bundle\FrameworkBundle\CacheWarmer\TranslationsCacheWarmer" public="false">
+            <argument type="service" id="translator" />
```

Comment:
translator.logging.inner does not exists in prod env. What should I do ? This is useful mainly in prod env.